dts/arm/st/f3/stm32f303Xb.dtsi

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023 Martin Gritzan
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/f3/stm32f303.dtsi>
+
+/ {
+	ccm0: memory@10000000 {
+		compatible = "zephyr,memory-region", "st,stm32-ccm";
+		reg = <0x10000000 DT_SIZE_K(8)>;
+		zephyr,memory-region = "CCM";
+	};
+
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(32)>;
+	};
+
+	soc {
+		flash-controller@40022000 {
+			flash0: flash@8000000 {
+				reg = <0x08000000 DT_SIZE_K(128)>;
+			};
+		};
+
+		dma2: dma@40020400 {
```

Comment:
I'm a bit puzzled by duplicated information here;
I'd suggest to get dts/arm/st/f3/stm32f303Xc.dtsi inluding dts/arm/st/f3/stm32f303Xb.dtsi

Reply:
Yep. I also thought about including another layer of inclusion that does all the duplicated stuff and have the ...Xb.dtsi and ...Xc.dtsi only define the RAM- and flash node, however, I did not come up with a good name for that other layer of inclusion. So as you said.